es6-shim.js

```diff
@@ -1438,7 +1438,7 @@
       if (number === 0) {
         return 32;
       }
-      return 32 - (number).toString(2).length;
+      return 31 - Math.floor(Math.log(number) * Math.LOG2E + (1 / 2 / 4294967296));
```

Comment:
Can `1 / 2 / 4294967296` be cached in a variable outside the shimmed function, named so as to describe what it represents?